### DISCUSSION TITLE
Consigo usar o prisma para modelar o Sqflite?

### DISCUSSION THREAD

**Discussion by walex36** [Flutterando/forum#53] [Discussões gerais]
Boa tarde Pessoal, estou com uma duvida eu consigo usar o prisma para modelar o sqflite, tentei começar a fazer mas tive problema em passar o caminho do banco para o prisma pq o banco está dentro do celular.

**Comment by eduardoflorence** [comment]
@walex36, não sei se conseguirá trabalhar com o SQ**F**Lite, pois na documentação do Prisma só cita o SQLite, mas o caminho do banco no SQFlite você acha desta forma:
 
```dart
import 'package:path/path.dart';
import 'package:sqflite/sqflite.dart';
...
var databasesPath = await getDatabasesPath();
String path = join(databasesPath, 'demo.db');
```

**Reply by walex36** [reply]
@eduardoflorence  Sim eu consegui pegar a cominho para o banco mas não consegui fazer o prisma acessar ele.
